packages/frameworks-sveltekit/package.json

```
@@ -27,7 +27,7 @@
     "build": "pnpm clean && pnpm providers && pnpm check && svelte-package",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
-    "clean": "rm -rf client.* index.* package src/lib/providers",
+    "clean": "rm -rf dist client.* index.* package src/lib/providers",
```

```suggestion
    "clean": "rm -rf dist src/lib/providers",
```
these are also no longer used